# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lftp/hotfix_ssl.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

ftpclass.cc: In member function 'virtual int Ftp::Do()':
ftpclass.cc:1333: error: 'class Ftp::Connection' has no member named 'prot'
make[1]: *** [ftpclass.lo] Error 1

--- ./src/ftpclass.cc.orig	2006-05-11 19:54:01.000000000 +0200
+++ ./src/ftpclass.cc	2006-05-11 19:55:25.000000000 +0200
@@ -1326,12 +1326,14 @@
 	 goto usual_return;
 
    pre_CONNECTED_STATE:
+#if USE_SSL
       if(ftps && (!proxy || conn->proxy_is_http))
       {
 	 conn->MakeSSLBuffers(hostname);
 	 const char *initial_prot=ResMgr::Query("ftps:initial-prot",hostname);
 	 conn->prot=initial_prot[0];
       }
+#endif
       if(use_telnet_iac)
 	 conn->InitTelnetLayer();
 
